### ISSUE TITLE
Support for etcd auth

### ISSUE THREAD

**Issue by ghost** [rabbitmq/rabbitmq-peer-discovery-etcd#6]
Currently there doesnt appear to be any support for etcd auth, which would be a nice to have. If there is support for it, then this ticket should track documentation. If there is documentation, this can be closed.

**Comment by michaelklishin** [comment]
Depends on #22 and the transition to the v3 protocol (API).
